aws/aws_test.go

```diff
@@ -24,22 +24,54 @@ func TestWriteToFile(t *testing.T) {
 	}
 
 	fn := "test_creds.txt"
-	p := "testprofile"
+	p := "expiredprofile"
 
 	// Write credentials
 	err := WriteToFile(&c, fn, p)
 	if err != nil {
 		t.Fatal("Could not write credentials to file: ", err)
 	}
 
+	// sleep so above credentials expire
```

Comment:
Cap it please ;-)

Why not write an expired timestamp to the file instead of writing `Now()` and sleeping? This delays **every** run of the unit tests by 2 seconds...